### ISSUE TITLE
iOS VoiceOver Rotor support

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#126376]
We should make sure that Flutter applications are able to make full use of the iOS VoiceOver Rotor and that all our built-in widgets fully support the features of the VoiceOver Rotor.

https://developer.apple.com/videos/play/wwdc2020/10116/
https://support.apple.com/en-us/HT204783
https://developer.apple.com/documentation/objectivec/nsobject/1649788-accessibilitycustomrotors
https://developer.apple.com/documentation/uikit/uiaccessibilitycustomrotor

The custom rotor provides user ways to navigate the screen. If you have 3 buttons and 3 text labels in a group, you can create a rotor to only traverse the buttons while skip the labels.

The goal is to figure out an API in flutter for people to create rotor in their iOS app. We also need to investigate whether there are similar APIs in other platforms.

**Comment by goderbauer** [comment]
@chunhtai I seem to recall that you did some work towards this. Do you know what the status is?

**Comment by chunhtai** [comment]
No I don't recall I have done anything in this space.

**Comment by TheQuinbox** [comment]
Its worth mentioning that Google's TalkBack now has a feature that's quite similar. This should be implemented as well.

**Comment by vick08** [comment]
It might be helpful to break this request into several issues. I think this issue is too generic and cannot be implemented as described
I will start a new issue on custom actions specifically, because there is a pretty good parity between iOS and Android as far as that feature is concerned. 
Unfortunately, the web is a bit more complicated, but we can handle that case separately.
Suffice it to say that adding custom actions support to Flutter would definitely help developers to make their apps more accessible. 
 , .

**Comment by goderbauer** [comment]
Is the rotor different from the radial context menu on iOS? We do support custom actions that - according to the docs - on iOS are surfaced in the "radial context menu" and on Android in the local context menu, see https://master-api.flutter.dev/flutter/semantics/CustomSemanticsAction-class.html.

Can someone clarify if that's exactly what this issue is asking for or specify how this is different / what is missing from our current support?
